### PR TITLE
update example orchestration to 5.x

### LINKED ISSUE #4435: update example orchestration to 5.x

### PR DESCRIPTION
**Pull request by geomonlin**
Fixes #4435 .

### CODE REVIEW

File: examples/sharding-jdbc-example/orchestration-example/orchestration-raw-jdbc-example/src/main/resources/META-INF/zookeeper/cloud/encrypt.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
-  overwrite: false
-  registry:
-    type: zookeeper
+  demo_yaml_ds_encrypt:
+    orchestrationType: config_center,registry_center
+    instanceType: zookeeper
     serverLists: localhost:2181
     namespace: orchestration-yaml-demo
+    props:
+      overwrite: true
```
Comment (dongzl) [inline examples/sharding-jdbc-example/orchestration-example/orchestration-raw-jdbc-example/src/main/resources/META-INF/zookeeper/cloud/encrypt.yaml]: There should be a blank line.

File: examples/sharding-jdbc-example/orchestration-example/orchestration-spring-boot-example/src/main/resources/application-local-nacos-encrypt.properties [hunk 2]
```diff
@@ -31,8 +31,14 @@ spring.shardingsphere.encrypt.tables.t_order.columns.status.encryptor=status_enc
...
+spring.shardingsphere.orchestration.demo_spring_boot_ds_encrypt.instance-type=nacos
+spring.shardingsphere.orchestration.demo_spring_boot_ds_encrypt.server-lists=localhost:8848
+spring.shardingsphere.orchestration.demo_spring_boot_ds_encrypt.namespace=
+spring.shardingsphere.orchestration.demo_spring_boot_ds_encrypt.props.overwrite=true
+
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_encrypt.orchestration-type=registry_center
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_encrypt.instance-type=zookeeper
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_encrypt.server-lists=localhost:2181
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_encrypt.namespace=orchestration-spring-boot-demo
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_encrypt.props.overwrite=true
```
Comment (dongzl) [inline examples/sharding-jdbc-example/orchestration-example/orchestration-spring-boot-example/src/main/resources/application-local-nacos-encrypt.properties]: There should be a blank line.

File: examples/sharding-jdbc-example/orchestration-example/orchestration-spring-boot-example/src/main/resources/application-local-nacos-master-slave.properties [hunk 3]
```diff
@@ -23,8 +23,14 @@ spring.shardingsphere.masterslave.name=ds_ms
...
+spring.shardingsphere.orchestration.demo_spring_boot_ds_ms.instance-type=nacos
+spring.shardingsphere.orchestration.demo_spring_boot_ds_ms.server-lists=localhost:8848
+spring.shardingsphere.orchestration.demo_spring_boot_ds_ms.namespace=
+spring.shardingsphere.orchestration.demo_spring_boot_ds_ms.props.overwrite=true
+
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_ms.orchestration-type=registry_center
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_ms.instance-type=zookeeper
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_ms.server-lists=localhost:2181
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_ms.namespace=orchestration-spring-boot-demo
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_ms.props.overwrite=true
```
Comment (dongzl) [inline examples/sharding-jdbc-example/orchestration-example/orchestration-spring-boot-example/src/main/resources/application-local-nacos-master-slave.properties]: The same to above.

File: examples/sharding-jdbc-example/orchestration-example/orchestration-spring-boot-example/src/main/resources/application-local-nacos-sharding-databases-tables.properties [hunk 4]
```diff
@@ -35,8 +35,15 @@ spring.shardingsphere.sharding.tables.t_order_item.key-generator.column=order_it
...
+spring.shardingsphere.orchestration.demo_spring_boot_ds_sharding.server-lists=localhost:8848
+spring.shardingsphere.orchestration.demo_spring_boot_ds_sharding.namespace=
+spring.shardingsphere.orchestration.demo_spring_boot_ds_sharding.props.overwrite=true
+
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_sharding.orchestration-type=registry_center
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_sharding.instance-type=zookeeper
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_sharding.server-lists=localhost:2181
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_sharding.namespace=orchestration-spring-boot-demo
+spring.shardingsphere.orchestration.demo1_spring_boot_ds_sharding.props.overwrite=true
+
```
Comment (dongzl) [inline examples/sharding-jdbc-example/orchestration-example/orchestration-spring-boot-example/src/main/resources/application-local-nacos-sharding-databases-tables.properties]: There are redundant blank line.

File: examples/sharding-jdbc-example/orchestration-example/orchestration-spring-namespace-example/src/main/resources/META-INF/zookeeper/local/application-encrypt.xml [hunk 5]
```diff
@@ -59,15 +59,15 @@
             </encrypt:tables>
             <encrypt:encryptors>
                 <encrypt:encryptor id="name_encryptor" type="AES" props-ref="props"/>
-                <encrypt:encryptor id="pwd_encryptor" type="assistedTest" />
```
Comment (dongzl) [inline examples/sharding-jdbc-example/orchestration-example/orchestration-spring-namespace-example/src/main/resources/META-INF/zookeeper/local/application-encrypt.xml]: Why fix this type param?

File: examples/sharding-jdbc-example/orchestration-example/orchestration-spring-namespace-example/src/main/resources/META-INF/zookeeper/local/application-master-slave.xml [hunk 6]
```diff
@@ -15,44 +15,44 @@
...
-    
+
     <bean id="demo_ds_slave_1" class="com.zaxxer.hikari.HikariDataSource" destroy-method="close">
         <property name="driverClassName" value="com.mysql.jdbc.Driver"/>
         <property name="jdbcUrl" value="jdbc:mysql://localhost:3306/demo_ds_slave_1"/>
         <property name="username" value="root"/>
         <property name="password" value=""/>
     </bean>
 
-    <master-slave:load-balance-algorithm id="randomStrategy" type="RANDOM" />
```
Comment (kimmking) [inline examples/sharding-jdbc-example/orchestration-example/orchestration-spring-namespace-example/src/main/resources/META-INF/zookeeper/local/application-master-slave.xml]: the last blank is standard.
should be retrieved.

File: examples/sharding-jdbc-example/orchestration-example/orchestration-raw-jdbc-example/src/main/resources/META-INF/nacos/local/encrypt.yaml [hunk 7]
```diff
@@ -3,7 +3,7 @@ orchestration:
     orchestrationType: config_center
     instanceType: nacos
     serverLists: localhost:8848
-    namespace: orchestration-yaml-demo
```
[the remark this answers is not in the document]
Comment (geomonlin) [reply examples/sharding-jdbc-example/orchestration-example/orchestration-raw-jdbc-example/src/main/resources/META-INF/nacos/local/encrypt.yaml]: This namespace  become namespaceId in naocs. If write namespace ,found should write  value of n**amespaceId "db5af0a7-ae65-49b2-810a-df5cb77103a1"**. However Show the configuration with empty namespace by default in nacos. 
image

File: examples/sharding-jdbc-example/orchestration-example/orchestration-raw-jdbc-example/src/main/resources/META-INF/nacos/cloud/encrypt.yaml [hunk 8]
```diff
@@ -1,10 +1,18 @@
...
-  name: demo_yaml_ds_encrypt
-  overwrite: false
-  registry:
-    type: nacos
+  demo_naocs_yaml_ds_encrypt:
+    orchestrationType: config_center
+    instanceType: nacos
     serverLists: localhost:8848
-    namespace:
+    namespace: orchestration-yaml-demo
```
[the remark this answers is not in the document]
Comment (geomonlin) [reply examples/sharding-jdbc-example/orchestration-example/orchestration-raw-jdbc-example/src/main/resources/META-INF/nacos/cloud/encrypt.yaml]: yeah You're right. thx

**Comment by kimmking** [comment]
image

```
examples\sharding-jdbc-example\orchestration-example\orchestration-raw-jdbc-example\src\main\java\org\apache\shardingsphere\example\orchestration\raw\jdbc\JavaConfigurationExampleMain.java
examples\sharding-jdbc-example\orchestration-example\orchestration-raw-jdbc-example\src\main\java\org\apache\shardingsphere\example\orchestration\raw\jdbc\YamlConfigurationExampleMain.java

```

**Pull request merged**